app/functions/formatDates.ts

```diff
@@ -0,0 +1,23 @@
+import moment from 'moment-timezone';
+
+const TIMEZONE = 'America/Vancouver';
+const FORMAT_TYPE = {
+  seconds: 'MMM D, YYYY hh:mm:ss A (z)',
+  minutes: 'MMM D, YYYY hh:mm A (z)',
+  hours: 'MMM D, YYYY hh A (z)',
+  days_numbered: 'DD-MM-YYYY',
+  days_string: 'MMM Do, YYYY',
+  database_timestamp: 'YYYY-MM-DDTHH:mm:ss'
```

Comment:
It doesn't look like `database_timestamp` and `hours` formats are currently used anywhere - maybe we should just start with the minimum required number of formats and add any others as they become necessary?